README.md

```diff
@@ -96,13 +96,13 @@ export default class EditorComponent extends Component {
           cellContent: 'block+',
           inlineBorderStyle: { width: '0.5px', color: '#CCD1D9' },
         }),
-        heading,
+        heading: heading(),
         blockquote,
         horizontal_rule,
         code_block,
         text,
         hard_break,
-        block_rdfa,
+        block_rdfa: block_rdfa(),
```

Comment:
Typo: This should be `blockRdfaWithConfig()`.

Reply:
Fixed